libs/pipelines/api_pipeline.py

```diff
@@ -218,6 +217,34 @@ def build_county_summary_from_model_output(input_dir) -> List[APIOutput]:
     return results
 
 
+def remove_root(obj: dict):
+    """Removes __root__ and replaces with __root__ value.
+
+    When pydantic models with the key __root__ are used to represent
+    json lists are serialized using `pydantic_model.dict()` the "__root__"
+    key is included. This will remove that __root__ key and replace with a list.
```

Comment:
Thank you for the comments, but I cannot parse the first sentence here. 😬 Mind rephrasing?

Reply:
heh yeah will do